Dockerfile.cockpit

```diff
@@ -5,5 +5,8 @@ RUN dnf install -y cockpit-ws cockpit-kubernetes && dnf clean all
 RUN echo $'[Negotiate]\nCommand = /usr/libexec/cockpit-stub\n[WebService]\nShell = /shell/simple.html' > /etc/cockpit/cockpit.conf
 RUN rm -rf /usr/share/cockpit/kubernetes/
 
+COPY welder-web*.rpm /tmp/
+RUN dnf -y install /tmp/welder-web*.rpm && rm -f /tmp/welder-web*.rpm
```

Comment:
I just realized that this won't actually save any space as it's already the COPY layer that has the file. But nevermind, it still saves an extra layer (even though not  the actual space for the .rpm, but that's small enough).